store/index.js

```diff
@@ -2,24 +2,28 @@
  * Developer : corner
  * Description : 해당 index 스토어
  ***********************/
-import api from '../plugins/api/index.js';
+import axios from "axios";
+import {api_url} from "~/plugins/api/config";
```

Comment:
Codacy has a fix for the issue: [Replace `api_url` with `·api_url·`](https://app.codacy.com/gh/Eight-Corner/front_toche_web/pullRequest?prid=9689045)
```suggestion
import { api_url } from "~/plugins/api/config";
```